conda/meta.yaml

```diff
@@ -1,28 +1,33 @@
 package:
   name: scwidgets
-  version: { { GIT_DESCRIBE_TAG } }
+  version: "1"
 
 source:
   path: ..
 
 requirements:
+  build:
```

Comment:
I don't think you need to specify build dependencies if there is actually no dependencies at build time.

Reply:
I think your right I'll remove.

Reply:
Having looked into it I currently add python as a build requirement so that the conda python folder which I move to in build.sh exists. I've tried creating and moving into a different folder but this seems not to work.